src/marketplace/ListingsView.js

```diff
@@ -31,28 +31,12 @@ class ListingsView extends React.Component<Props, State> {
 	}
 
 	//When sellerType is updated we need to get the new sells
-	componentDidUpdate(prevProps:Props) : void {
-		this.directSaleToProperFunction();
+	componentDidUpdate(prevProps:Props, prevState:State) : void {
```

Comment:
Spacing should be:
```componentDidUpdate(prevProps: Props, prevState: State): void {```
Namely:
- no space after the close paren
- space after both colons